TACC/WMA-Tapis-Templates
# task/WA-310: MATLAB Interactive on S3

## Overview: ##
Moves MATLAB DS app to Stampede3

## PR Status: ##


## Related Jira tickets: ##

* [WA-310](https://tacc-main.atlassian.net/browse/WA-310)

## Summary of Changes: ##
- Modified `matlab` HPC app definition to match Stampede3 hardware parameters
- Updated descriptions + version number to reflect updated version available on S3 (matlab r2023b)
- Updated app profile to not load swr, which is not installed on S3

## Testing Steps: ##
- App is up on designsafe dev: https://designsafeci-dev.tacc.utexas.edu/workspace/matlab?appVersion=9.15.0
- Ensure that, upon successfully submitting app form, job queues and eventually runs
- Log in to DCV session once available and ensure matlab starts up and is pointing to your specified working directory
- Manually close matlab session within DCV session and confirm DCV session ends (I've observed the window doesn't close, but the session is ended; verified by reloading window and re-attempting login and attempting to connect again via portal 'Open Session' button)


## UI Photos:

## Notes: ##
